blog/2022-09-24-LFX-Blog-Carol.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: Hygienic error handling and validation for pipelines
+author: Carol Geng
+author_title: Tremor 2022 Summer Mentee
+author_url: https://www.linkedin.com/in/carolgeng/
+tags: ["cncf", "mentorship"]
+draft: false
+description: "Carol’s Tremor Mentorship Report"
+---
+
+# Introduction
```

Comment:
In the deploy it just shows "Introduction" as the title of the blog post. I think the title should be the above.

Suggestion: Add the title above with one `#` before it, so it becomes the visible title and prepend another point to all the headings in the text.

```suggestion
# Hygienic error handling and validation for pipelines

## Introduction
```